sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -46,6 +46,7 @@
                     force_save="1"
                     context="{'partner_id':order_partner_id, 'quantity':product_uom_qty, 'pricelist':pricelist_id, 'uom':product_uom, 'company_id': company_id}"
                 />
+                <field name="product_uom_category_id" invisible="1" />
```

Comment:
This view should have at least the same fields and options as the original v13 view:
https://github.com/odoo/odoo/blob/13.0/addons/sale/views/sale_views.xml#L436-L553
We should compare and add the missing fields and options.
It would be nice if all the fields have the attribute `optional="show/hide"`. (Except key fields like partner or product)